src/containers/NamespacesDropdown/NamespacesDropdown.js

```diff
@@ -17,7 +17,11 @@ import { connect } from 'react-redux';
 import { ALL_NAMESPACES } from '@tektoncd/dashboard-utils';
```

Comment:
Do we need to make changes to the NamespacesDropdown? If we're only giving access to a single namespace a Dropdown probably doesn't make sense in most cases, and would either be hidden or replaced with text showing the tenant namespace.

If we do need to display a list of namespaces (only one in this case), would it make more sense to handle this in the `getNamespaces` selector so it's transparent to most consumers? There should be very few places in the code that need to be aware of the tenant namespace specifically.

Reply:
Good point, i will look at it tomorrow.

Reply:
@AlanGreene Is it ok if i improve that in a follow up PR ?